docs/learn/dealing-with-latency.md

```diff
@@ -279,6 +279,11 @@ Advanced games will have most of their world predicted, allowing the client and
 There's no prediction implementation right now in Netcode for GameObjects, but you can implement your own. See our [roadmap](https://unity.com/roadmap/unity-platform/multiplayer-networking) for more information.
 :::
 
+### Client Side Reinterpretation (state solving)
+For a few reasons, clients can't always stay in sync tick by tick, frame by frame. It's possible to adapt your gameplay to allow your clients to be desynced for a few seconds, as long as they sync back again eventually.
+This is usually the case for death ragdolls in games where a dead character's body doesn't influence your gameplay anymore and so can be desynced from one client the another. The server will sync to all clients "this character is now dead" and will send it's death position. From then on, clients can apply all the ragdol physics they want with no need of syncing it between clients (as it has no gameplay impact, we don't care if I see the ragdol in a tree and you see the ragdol in a ditch).
```

Comment:
*ragdoll (not ragdol)

Reply:
ah woops, thanks for the catch. This is mostly a draft, more polishing and details to come.